src/reducers/Transactions.js

```diff
@@ -3,33 +3,32 @@ import { TransactionConstants } from '../actions/Transaction'
 const initialState = []
 
 export default function Transactions(state = initialState, action = {}) {
-    switch (action.type) {
+  switch (action.type) {
```

Comment:
Just indentation fixes here 👌

Reply:
Yeah I figured it. Github show some weird code diff here